matter_server/client/__init__.py

```diff
@@ -1 +1,2 @@
-from .client import MatterClient
+"""Client for the MatterServer."""
+from .client import MatterClient  # noqa
```

Comment:
Define `__all__` instead of disabling flake8.

Reply:
valid point, pr submitted